src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -182,9 +185,14 @@
 	private Environment env;
 
 	/**
-	 * Printer helper.
+	 * Token detector, which delegates tokens to {@link PrinterTokenWriter}
+	 */
+	private PrinterTokenDetector printer;
```

Comment:
what about adding Helper in the class name of PrinterTokenDetector to make the intention clear?

Reply:
Good idea. I will rename it.